docs/configuration/http_filters/grpc_json_transcoding_filter.rst

```diff
@@ -0,0 +1,19 @@
+.. _config_http_filters_grpc_json_transcoding:
+
+gRPC-JSON transcoding filter
+============================
+
+gRPC :ref:`architecture overview <arch_overview_grpc>`.
+
+This is a filter which enables the bridging of a JSON-REST client to a gRPC server.
+
+.. code-block:: json
+
+  {
+    "type": "both",
+    "name": "grpc_json_transcoding",
+    "config": {
+      "proto_descriptors": "proto.pb",
```

Comment:
I believe it is possible with grpc reflection, however we have no experience with that and reflection itself is still v1alpha. I will take this on later task, not in this PR. predefined pb doesn't provide better error checking.